core/src/com/unciv/logic/city/CityStats.kt

```diff
@@ -357,6 +357,7 @@ class CityStats {
         return construction.name == filter
                 || (filter == "land units" && construction is BaseUnit && construction.unitType.isLandUnit())
                 || (filter == "naval units" && construction is BaseUnit && construction.unitType.isWaterUnit())
+                || (filter == "ranged units" && construction is BaseUnit && construction.unitType == UnitType.Ranged)
```

Comment:
Here we have a definition problem, and I blame Civ V.
You see, Ranged (like Melee) is both a Unit Type, and a class of UnitTypes.
When I say 'ranged units', do I mean also Siege units? Ranged naval units?
Below, 'melee units' is translated as the *class* of unittypes, and here it's the specific unit type... maybe we should change the terminology to 'close-range' and 'long-range'? But then what about Machine Guns, which are ranged but have 1 range? Or change the name of the unitType? I don't know :/